src/entities/session/model/store.ts

```diff
@@ -0,0 +1,26 @@
+import { createDomain, createEffect, sample } from "effector";
+import { login, logout, getAuthStatus, getCurrentUser } from "./api";
+import { User } from "./types";
+
+const domain = createDomain("entities/session");
+
+export const onLogin = domain.event();
+export const onLogout = domain.event();
+export const onAppLoaded = domain.event();
+
+export const loginFx = createEffect(login);
+export const logoutFx = createEffect(logout);
+export const isAuthenticatedFx = createEffect(getAuthStatus);
+export const getUserFx = createEffect(getCurrentUser);
+
+sample({ clock: onLogin, target: loginFx });
```

Comment:
following the correct order - events > effects > stores > samples

Reply:
changed to correct order